fbench/src/fbench/client.cpp

```diff
@@ -220,8 +224,15 @@ Client::run()
                 strcat(_linebuf, _args->_queryStringToAppend.c_str());
             }
             int cLen = _args->_usePostMode ? urlSource.nextContent() : 0;
+            auto content = urlSource.content();
+            if (_args->_usePostMode && _args->_base64Decode) {
+                auto base64_decoded = Base64::decode(std::string(content, cLen));
+                content = base64_decoded.c_str();
```

Comment:
Here you have a lifetime issue. **base64_decoded** must live at least as long as **content**.

Reply:
Java developer negligence... fixed your other comments as comments as well.